community/dnscrypt-proxy/config-full-paths.patch

```diff
@@ -452,7 +492,7 @@ index 0000000..347ada5
 +
 +  [sources.'public-resolvers']
 +  urls = ['https://raw.githubusercontent.com/DNSCrypt/dnscrypt-resolvers/master/v2/public-resolvers.md', 'https://download.dnscrypt.info/resolvers-list/v2/public-resolvers.md']
-+  cache_file = '/var/cache/dnscrypt-proxy/public-resolvers.md'
++  cache_file = 'public-resolvers.md'
```

Comment:
Where is this file downloaded? There's a reason why the full path has been removed?

Reply:
My error - will fix